Source/WebCore/platform/graphics/skia/GraphicsContextSkia.cpp

```diff
@@ -895,12 +898,9 @@ void GraphicsContextSkia::drawPattern(NativeImage& nativeImage, const FloatRect&
     if (spacing.isZero() && !needsClip)
```

Comment:
Here we could just do `if (spacing.isZero() && tileRect.size() == nativeImage.size())`